fact/src/host_info.rs

```diff
@@ -101,9 +101,20 @@ pub fn get_username(uid: u32) -> &'static str {
     }
 }
 
-pub fn get_mount_ns(pid: &str) -> u64 {
+// get_mount_ns
+//
+// Returns the mount namesace identifier for a specified PID. If the host flag
+// is set, the PID will be interpreted as a process inside the host pid
+// namespace, otherwise as a process inside the current container pid namespace.
```

Comment:
[nit] Docstrings use `///` in Rust
```suggestion
/// get_mount_ns
///
/// Returns the mount namesace identifier for a specified PID. If the host flag
/// is set, the PID will be interpreted as a process inside the host pid
/// namespace, otherwise as a process inside the current container pid namespace.
```